axolotl-web/src/components/AddGroupMembersModal.vue

```diff
@@ -96,19 +112,40 @@ export default {
   },
   computed: {
     contactsFilterd () {
-      return this.$store.state.contactsFilterd
+      return this.$store.state.contactsFilterd.filter(c => {
+        if(c.UUID[0]==0&&c.UUID[c.UUID.length-1]==0) return true;
+        var found = this.allreadyAdded.find(function(element) {
+            return element.Tel == c.Tel;
+        });
+        if(typeof found =="undefined")return true;
+        else return false;
+      }); 
     },
   },
   watch:{
     allreadyAdded(){
       var that = this
-      this.contacts=that.$store.state.contacts.filter(function (el) {
```

Comment:
So many copy pasted lines here, + a very bad indentation, please refactor that part...